routes/app/feed/index.js

```diff
@@ -1,13 +1,15 @@
 const { Router }  = require('express');
 
-
+const User        = require('../../../models/user');
 const router    = new Router();
 module.exports  = router;
 
 
-router.get('/feed', function(req, res) {
+router.get('/feed', User.authUser(), function(req, res) {
```

Comment:
Ohhhhh I like the middleware! There's an easier way to set it for all routes on a router instead of adding it to each individual route (this works for this specific case). In the `routes/app` folder, if there is more than one route in one of the `index.js` files, put this at the top:
```
router.use('/friend*', User.authUser());
```
So this middleware would be used for anything that starts with `/friend` (ex. `/friends` or `/friend/1234`)